api/server/models/user.js

```diff
@@ -37,10 +36,18 @@ module.exports = function (User) {
                                         if (r_verification_status == 0) {
                                             callback(null, 0, 'Please verify you account first', {});
                                         } else {
-                                            var data = {
-                                                user_id: result.id
-                                            };
-                                            callback(null, 1, 'Success login', data);
+                                            //-START--get access token---------
+                                            User.login({
```

Comment:
do we need to do "UTIL.match_encode_password" above. 
since User.login will go into err if password is not correct. 
i dont think above password matching is required